lib/lazy_observers/rails_observer.rb

```
@@ -0,0 +1,13 @@
+if ActiveRecord::VERSION::MAJOR < 4
+  require 'active_record/observer'
+else
+  begin
+    require 'rails/observers/activerecord/active_record'
+  rescue LoadError => _
+    $stderr.puts(<<-MSG)
```

Hmm I like the message, but maybe raise instead of warn ?